### ISSUE TITLE
I am notified my browser is out of date even though I am using the latest Chrome (54) on Macintosh

### ISSUE THREAD

**Issue by SkierX9** [browser-update/browser-update#415]
I often see a message that my browser is out of date - I don't recall which site I saw it on, but I do see that even when I go to www.github.com, I get a message that GitHub no longer supports my browser (though that message looks a little different than the standard browser out-of-date message I get that I think comes via browser-update.)

I now see that if I go to https://brower-update.org/update-browser.html, I get a message that my browser is out-of-date.  FYI, I am using Chrome Version 54.0.2840.71 on Macintosh.  When I go to About on Chrome, it tells me that Google Chrome is up to date.

Perhaps the issue comes from the fact that I am using OS X 10.10.5, because I am using a Mac Pro from early 2008, which cannot update to the latest Mac OS X?  Perhaps Chrome realizes this and tells me I am up-to-date even though there may be a newer version of Chrome for more recent versions of OS X?  I am just guessing here, because it seems tough to believe that broswer-update doesn't know what the latest version of Chrome is on Macintosh.

In any case, just thought I would mention it as it is a bit tiring seeing these notices and I am about to turn them off.

Thanks for looking into this.

**Comment by SkierX9** [comment]
Hello-

Thanks for your reply.  Please find the attached JPEGs to see the
screenshots of those 2 links.


[image: test1.jpg]
[image: bar1.jpg]

**Comment by josselex** [comment]
Sorry, I can't see the images.  You can also just copy the text here.

**Comment by SkierX9** [comment]
You cannot view standard .jpg images?  Is there a reason why?  I see the
files attached just fine and GMail displays the images.

In any case, the bar.html page has a header saying :

*Your web browser (Chrome 54) is out of date.* Update your browser for more
security, speed and the best experience on this site.Update browser
<https://browser-update.org/update-browser.html?force_outdated=true#3.2.1:browser-update.org>
 Ignore
Browser Notification Debug-Mode (v3.2.1)
Browser would normally be notified: true
*Reasons to show*: below required
*Browser info*
is_latest:false, is_insecure:true, other:false, no_device_update:false, cookie
set:true

Then below the header it says:

Testing the bar
 Clear cookie before showing
Detection
User-Agent-String (Javascript): Mozilla/5.0 (Macintosh; Intel Mac OS X
10_10_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/54.0.2840.71
Safari/537.36
Detected as (Javascript): Chrome 54
Detected Language (Javascript): enOptions:
{"test":true,"jsshowurl":"../update.show.js","domain":"","llfull":"en-us","ll":"en","apiver":-1,"jsv":"3.2.1","required":{"e":14,"i":11,"f":58,"o":50,"o_a":42.1,"s":10.1,"c":65,"a":535,"y":18.6,"v":1.12,"uc":12.1,"samsung":6.1,"ios":9},"reminder":24,"reminderClosed":168,"pageurl":"
browser-update.org","newwindow":true,"reasons":["below
required"],"hide_reasons":[],"is_below_required":true,"notified":true,"already_shown":false}



______

On the test.html page it has a header that says:

Browser-Update.org <http://browser-update.org/>
An initiative by websites to inform users to update their web browser

There are some links to the right of that "About the Project", etc....
Pretty standard.

And the body is:

Identification

User-Agent-String (HTTP Header):Mozilla/5.0 (Macintosh; Intel Mac OS X
10_10_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/54.0.2840.71
Safari/537.36

User-Agent-String (Javascript):Mozilla/5.0 (Macintosh; Intel Mac OS X
10_10_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/54.0.2840.71
Safari/537.36
Detection

Detected Browser as (Javascript):Chrome 54

Browser (Header):Chrome 54

System (Header):MacOS 10.10
Language

Lang (HTTP Header):en-US,en;q=0.8

Lang (Javascript):en
More{"ip3": "162.158.63.4", "cont": "NA", "ip": "127.0.0.1", "country":
"US"}

_________________________

Thanks again for your help.  Please let me know if you need anything else
from me.


Best regards,
Jeff

**Comment by josselex** [comment]
Thanks.

Something is strange.
If you really have OSX10.10 you should be able to upgrade to the latest
chrome
see
https://support.google.com/chrome/a/answer/7100626?hl=en
https://en.wikipedia.org/wiki/Google_Chrome#Compatibility

You should just go to
https://www.google.com/chrome/
and download the latest version.
Sometime the auto-update gets stuck.

Please tell me if that solved your problem.

Am Fr., 14. Sep. 2018 um 12:27 Uhr schrieb SkierX9 <notifications@github.com
>:

> You cannot view standard .jpg images? Is there a reason why? I see the
> files attached just fine and GMail displays the images.
>
> In any case, the bar.html page has a header saying :
>
> *Your web browser (Chrome 54) is out of date.* Update your browser for more
> security, speed and the best experience on this site.Update browser
> <
> https://browser-update.org/update-browser.html?force_outdated=true#3.2.1:browser-update.org
> >
> Ignore
> Browser Notification Debug-Mode (v3.2.1)
> Browser would normally be notified: true
> *Reasons to show*: below required
> *Browser info*
> is_latest:false, is_insecure:true, other:false, no_device_update:false,
> cookie
> set:true
>
> Then below the header it says:
>
> Testing the bar
> Clear cookie before showing
> Detection
> User-Agent-String (Javascript): Mozilla/5.0 (Macintosh; Intel Mac OS X
> 10_10_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/54.0.2840.71
> Safari/537.36
> Detected as (Javascript): Chrome 54
> Detected Language (Javascript): enOptions:
>
> {"test":true,"jsshowurl":"../update.show.js","domain":"","llfull":"en-us","ll":"en","apiver":-1,"jsv":"3.2.1","required":{"e":14,"i":11,"f":58,"o":50,"o_a":42.1,"s":10.1,"c":65,"a":535,"y":18.6,"v":1.12,"uc":12.1,"samsung":6.1,"ios":9},"reminder":24,"reminderClosed":168,"pageurl":"
> browser-update.org","newwindow":true,"reasons":["below
>
> required"],"hide_reasons":[],"is_below_required":true,"notified":true,"already_shown":false}
>
>
>
> ______
>
> On the test.html page it has a header that says:
>
> Browser-Update.org <http://browser-update.org/>
> An initiative by websites to inform users to update their web browser
>
> There are some links to the right of that "About the Project", etc....
> Pretty standard.
>
> And the body is:
>
> Identification
>
> User-Agent-String (HTTP Header):Mozilla/5.0 (Macintosh; Intel Mac OS X
> 10_10_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/54.0.2840.71
> Safari/537.36
>
> User-Agent-String (Javascript):Mozilla/5.0 (Macintosh; Intel Mac OS X
> 10_10_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/54.0.2840.71
> Safari/537.36
> Detection
>
> Detected Browser as (Javascript):Chrome 54
>
> Browser (Header):Chrome 54
>
> System (Header):MacOS 10.10
> Language
>
> Lang (HTTP Header):en-US,en;q=0.8
>
> Lang (Javascript):en
> More{"ip3": "162.158.63.4", "cont": "NA", "ip": "127.0.0.1", "country":
> "US"}
>
> _________________________
>
> Thanks again for your help. Please let me know if you need anything else
> from me.
>
>
> Best regards,
> Jeff
>
>
>
>
>
>
>
>

**Comment by grahamperrin** [comment]
> … a Mac Pro from early 2008, …

Please, what's the model identifier? And the graphics hardware? You'll find these things in _System Information_.

**Comment by josselex** [comment]
No further information... updating seems to have worked now? 
closing.

**Comment by SkierX9** [comment]
Sorry, I didn't see your message until now.

Model Identifier: MacPro 3,1
Graphics Hardware: NVIDIA GeForce 8800 GT w/512MB VRAM

Apparently this is a known issue for Google - earlier versions of Chrome
(or the Chrome Updater) had some sort of bug which was updated in later
versions - however, since my version never auto-updated, I never received
the newer versions.  Now I downloaded the latest version manually, so
hopefully I will be all set going forward.

https://bugs.chromium.org/p/chromium/issues/detail?id=662228

Thanks for your help.
